edb/edgeql/compiler/casts.py

```diff
@@ -69,7 +69,7 @@ def compile_cast(
             ctx=ctx,
             srcctx=ir_expr.context)
 
-    elif irutils.is_untyped_empty_array_expr(ir_expr):
+    if irutils.is_untyped_empty_array_expr(ir_expr):
```

Comment:
Why change all the elif to if?

Reply:
I find it more readable, because in this case I process this as guard statement - just a special case that does an early return.

I know that having `elif` results in same control flow, but I find them unnecessary.

Reply:
(Which definitely doesn't always make it worth doing this sort of thing, but in this kind of long function with a lot of cases, I think I like it)